### PR TITLE
linux-fresh: update base image to use Qt 5.15

### PR DESCRIPTION
**Pull request by liushuyu**
- `linux-fresh`: update ~~base image to Ubuntu 20.04~~ Qt 5.15
- Other images: replaced `MAINTAINER` with `LABEL` (new specification)

### CODE REVIEW

File: linux-fresh/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
-FROM ubuntu:18.04
-MAINTAINER yuzu
+FROM ubuntu:20.04
```
Comment (lat9nq) [inline linux-fresh/Dockerfile:1]: It is 18.04 currently for compatibility with distros that old, and if we ever want to release on AppImageHub, we need to use 18.04.

File: linux-fresh/Dockerfile [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+LABEL maintainer="yuzu"
 
 ENV CLANG_VER=12
-ENV CMAKE_VER=3.16.3
+ENV CMAKE_VER=3.20.5
```
Comment (lat9nq) [inline linux-fresh/Dockerfile:5]: yuzu requires CMake 3.15. We should be testing against 3.15, not anything newer (thus 3.16 is wrong, too).

File: linux-mingw/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 FROM archlinux:latest
-MAINTAINER yuzu
+LABEL maintainer="yuzu"
```
Comment (lat9nq) [inline linux-mingw/Dockerfile:2]: Is it possible we can hold off on updating the MinGW container right now? I've been working on updating it with compatible packages -- the Qt5 packages it uses have upgraded and now cause issues in yuzu.

**Comment by liushuyu** [comment]
Partially reverted changes: `linux-fresh` will still use Ubuntu 18.04 as the base but use Qt 5.15 (contains security fixes).

**Comment by liushuyu** [comment]
This pull request is no longer relevant and should be closed.

**Pull request closed without merging**